Tree/LargestBST.cpp

```diff
@@ -0,0 +1,107 @@
+/*Find the BST with the maximum heigth i.e. largest BST of the binary tree, which the user will input.*/
+
+#include <iostream>
+#include <queue>
+#include<limits.h>
+
+template <typename T> // template to take any data type string and int as integer
+class BinaryTreeNode {
+   public:
+    T data;
+    BinaryTreeNode<T>* left;
+    BinaryTreeNode<T>* right;
+
+    BinaryTreeNode(T data) {  //constructor
+        this->data = data; 
+        left = NULL;
+        right = NULL;
+    }
+    ~BinaryTreeNode() {     //destructor
+        if (left) delete left;
+        if (right) delete right;
+    }
+};
+
+using namespace std;
+
+
+int height(BinaryTreeNode<int> *root)  //returnning heigth of the tree
```

Comment:
this is just the information about what it returns or what the function do, but how it does it is missing.